qiita_db/metadata_template.py

```diff
@@ -2257,7 +2270,8 @@ def load_template_to_dataframe(fn, strip_whitespace=True):
     # comment:
     #   using the tab character as "comment" we remove rows that are
     #   constituted only by delimiters i. e. empty rows.
-    template = pd.read_csv(fn, sep='\t', infer_datetime_format=True,
+    template = pd.read_csv(StringIO(''.join(holdfile)), sep='\t',
```

Comment:
actually, you only need to join with `'\n'` if you implement suggestion above to leave argument to `strip` empty